source/multiple-mongosyncs.txt

```diff
@@ -291,6 +277,9 @@ acts as the source cluster:
 - If you have not already done so, issue the :ref:`commit
   <c2c-api-commit>` command to each ``mongosync`` instance and wait
   until all of the commits to finish.
+  - To check if the sync process has been committed, issue the :ref:`progress 
```

Comment:
I think this new sentence isn't a bullet on the UI? And the :ref:progress

< c2c-api-progress > is appearing as is rather than as a variable hyperlink

Reply:
Opted to move the sentence just into the first bullet as originally asked.